### ISSUE TITLE
Won't Install Any More

### ISSUE THREAD

**Issue by madranet** [vlang/ui#136]
Just tried to update today with `v up` followed by `v install ui` and got the following:

```shell
v install ui
0

Error in decode() for Mod error_ptr=: %s
0


<style>
...
massive snip
...
</div>


0

Error in decode() for Mod error_ptr=: %s
Errors while retrieving meta data for module ui:
Skipping module "ui", since it is missing name or url information.
Skipping module "ui", since it is missing name or url information.
```

The  massive chunk of CSS and HTML I've omitted seems to be the source code of the V Modules page. 

Output HTML-peviewed in Textmate:

![SS 2020-05-19 at 16 54 09](https://user-images.githubusercontent.com/669658/82348964-693d1400-99f1-11ea-803b-bbda8ec87bc9.png)

**Comment by dexbyte** [comment]
For now just clone this repository to the `.vmodules` folder on your mac

**Comment by madranet** [comment]
It's OK. Got it sorted in the end via the Telegram channel. I should have updated the issue.  

The problem was caused by a recent change to the update command which was incompatible with the old version of the `ui` module I had installed previously.  

The answer is to `rm` the existing  `ui` module directory first and then run the update & install again:

```
$> sudo rm -r ~/.vmodules/ui
$>  v up
$:> v install ui
```

does the trick.
